src/AgentContainer.cpp

```diff
@@ -63,8 +63,11 @@ AgentContainer::AgentContainer (const amrex::Geometry            & a_geom,  /*!<
         amrex::ParmParse pp("agent");
         pp.query("shelter_compliance", m_shelter_compliance);
         pp.query("symptomatic_withdraw_compliance", m_symptomatic_withdraw_compliance);
-        queryArray(pp, "student_teacher_ratio", m_student_teacher_ratio, SchoolType::total);
-
+        int stratio[SchoolType::total];
+        queryArray(pp, "student_teacher_ratio", stratio, SchoolType::total);
+        for (unsigned int i = 0; i < SchoolType::total; ++i) {
+            m_student_teacher_ratio[i] = stratio[i];
+        }
```

Comment:
Will this block of code work on a GPU always? `stratio` seems to be a CPU array while `m_student_teacher_ratio` is a GPU array?

Reply:
`amrex::GPUArray` lives on the host but can be captured by value in device lambda functions (unlike the C array that is being replaced here)